.babelrc

```diff
@@ -51,7 +51,7 @@
       {
         "root": "./src",
         "alias": {
-          "@@vet360": "./src/platform/user/profile/vet360",
+          "@@vet360": "./src/platform/user/profile/vap-svc",
```

Comment:
A future PR will rename this alias from `@@vet360` to `@@vap-svc`. Renaming this alias now would make all of the simple file-renames in this PR more complex because their content would have changed as well. I wanted to keep this PR as simple as possible since it's already touching so many files.

Reply:
`svc` = `service`?

Reply:
yeah that's the idea.